feat: affiche une barre de statistique dans le pied de page

ggrossetie opened the pull request:
resolves #1087

ggrossetie committed:
feat: affiche une barre de statistique dans le pied de page

RochDLY commented:
Je remets également pour cette PR ce dont nous avons discuté en privé : la barre des statistiques en bas est très bien, par contre nous nous sommes demandés si le scrool pour afficher le footer n'était pas dérangeant. Si oui, est-ce qu'il est possible de mettre les informations du footer ailleurs dans la page (par exemple dans le bandeau noir en en-tête, afin de supprimer ce footer ? 

ggrossetie commented:
> Je remets également pour cette PR ce dont nous avons discuté en privé : la barre des statistiques en bas est très bien, par contre nous nous sommes demandés si le scrool pour afficher le footer n'était pas dérangeant. Si oui, est-ce qu'il est possible de mettre les informations du footer ailleurs dans la page (par exemple dans le bandeau noir en en-tête, afin de supprimer ce footer ? 

Comme ce n'est pas directement lié, j'ai crée une discussion sur le sujet : https://github.com/EcrituresNumeriques/stylo/discussions/1095

Est-ce qu'on déploie ce changement sur stylo-dev afin de collecter des retours supplémentaires ?

ggrossetie commented:
Nous avons décidé de supprimer le pied de page sur la page de l'éditeur de texte afin d'éviter d'avoir une barre de scroll sur la fenêtre.

ggrossetie committed:
feat: supprime le pied de page sur les pages /articles

- crée des modules CSS pour chaque composant
- supprime les styles non utilisés
- supprime la barre de scroll verticale sur l'éditeur

ggrossetie committed:
fix: supprime le style global sur body
